cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -980,16 +981,25 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
           p(
             """The name-globbing for procedure and user defined function names is a simplified version of globbing for filename expansions, only allowing two wildcard characters -- `+*+` and `?`.
               |They are used for multiple and single character matches, where `+*+` means 0 or more characters and `?` matches exactly one character.""".stripMargin)
+          note(
+            """The name-globbing is subject to the <<cypher-manual#cypher-naming, standard Cypher restrictions on valid identifiers>>,
```

Comment:
In the section about identifier restrictions (https://neo4j.com/docs/cypher-manual/4.2/syntax/naming/) it is mentioned that database names are an exception as they allow dots to not need escaping, should we also update to mention globbing there?

Reply:
I'm a bit unsure, as I think database names are much more commonly used than the current globbing. You could add a small amendment to the note about globbing if you like, but you could also wait until we do proper globbing and have an own section in docs for it in 5.x

Reply:
Since it hasn't been there before now I think its better to wait until it is more used then (cause I was leaning towards not but wanted a second opinion)